### PR TITLE
{lib}[GCCcore/8.2.0] LibSoup v2.66.1

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Depends on ~~#8113~~

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/f864ee291907d1309e216fcdf3e68948 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/10895abfc5ee63f71b4030d539f5d1ee for a full test report.

**Comment by boegel** [comment]
@akesandgren Adding

```python
toolchainopts = {'cstd': 'gnu89'}
```

fixes the issue for me (inspired by https://bugs.freedesktop.org/show_bug.cgi?id=101192)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/91598dd5d85be785dd56edbad6da63f9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/5cec35dce93287bc544d3a07d8a29bec for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dee9db2f8618d2361364b58faa2538c0 for a full test report.

File: easybuild/easyconfigs/l/LibSoup/LibSoup-2.66.1-GCCcore-8.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+dependencies = [
+    ('libxml2', '2.9.8'),
+    ('SQLite', '3.27.2'),
+    ('GLib', '2.60.1'),
+    ('libpsl', '0.21.0'),
+    ('cURL', '7.63.0'),
+]
+
+configopts = '--libdir lib -Dgssapi=false -Dvapi=false '
```
Comment (boegel) [inline easybuild/easyconfigs/l/LibSoup/LibSoup-2.66.1-GCCcore-8.2.0.eb]: @akesandgren You can drop the `--libdir lib` thanks to https://github.com/easybuilders/easybuild-easyblocks/pull/1673

File: easybuild/easyconfigs/l/LibSoup/LibSoup-2.66.1-GCCcore-8.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+modextrapaths = {
+    'XDG_DATA_DIRS': 'share',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['include', 'lib']
```
Comment (boegel) [inline easybuild/easyconfigs/l/LibSoup/LibSoup-2.66.1-GCCcore-8.2.0.eb]: @akesandgren We should make this a bit stricter imho...

```python
sanity_check_paths = {
    'files': ['lib/libsoup-2.4.%s' % SHLIB_EXT, 'lib/libsoup-gnome-2.4.%s' % SHLIB_EXT],
    'dirs': ['include/libsoup-2.4/libsoup', 'include/libsoup-gnome-2.4/libsoup', 'lib/pkgconfig'],
}

Comment (akesandgren) [reply easybuild/easyconfigs/l/LibSoup/LibSoup-2.66.1-GCCcore-8.2.0.eb]: Ah yes.... I forgot that. I just put some place holders there while working on it, and then completely forgot.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3110.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/f5355e9124bc7e90efcbb04cda95b4ec for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2039.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/eaacfc99bba2c51a6dbd3350eddba3f3 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2608.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1f99ef55d9189f0c8766f2012ecb0e2c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**